src/index.ts

```diff
@@ -10,25 +10,25 @@ import {pathToIndices} from "./utils";
  * @param entropy optional additional entropy
```

Comment:
can we export `deriveChildSK` and `deriveMasterSK`?

Reply:
I would rather export just deriveKey(seed, path) instead of some internal methods that aren't suppose to be used directly :)

Reply:
Ok, it would be nice to have an exported function that can derive the same key given a seed/entropy. Right now, this function will always return new random keys b/c the param is just _additional_ entropy.